include/coreir/ir/dynamic_bit_vector.h

```diff
@@ -309,7 +308,6 @@ namespace bsim {
       ind++;
 
       char format = str_raw[ind];
-      (void) format;
```

Comment:
Without this, it will fail on Linux citing an unused variable warning.

Reply:
Strange, isn't it used in the assertion below? Also seems, to compile on travis